### ISSUE TITLE
Web: Upgrading from 44.0.0 to 44.0.1 results in error: export 'getManifest2' was not found in './PlatformUtils'

### ISSUE THREAD

**Issue by pors** [expo/expo#15714]
### Summary

Upgrading from 44.0.0 to 44.0.1 results in this error when building for web (in development):

```
./node_modules/expo-asset/build/AssetSources.js:78:20
"export 'getManifest2' was not found in './PlatformUtils'
  76 |
  77 |   // For assets during development using manifest2, we use the development server's URL origin
> 78 |   const manifest2 = getManifest2();
     |                    ^
  79 |
  80 |   if (manifest2?.extra?.expoGo?.developer) {
  81 |     const baseUrl = new URL(`http://${manifest2.extra.expoGo.debuggerHost}`);
```

### Managed or bare workflow? If you have `ios/` or `android/` directories in your project, the answer is bare!

managed

### What platform(s) does this occur on?

Web

### SDK Version (managed workflow only)

44

### Environment

```
  Expo CLI 5.0.3 environment info:
    System:
      OS: macOS 12.0.1
      Shell: 5.8 - /bin/zsh
    Binaries:
      Node: 16.0.0 - ~/.nvm/versions/node/v16.0.0/bin/node
      Yarn: 1.22.10 - ~/.nvm/versions/node/v16.0.0/bin/yarn
      npm: 8.2.0 - ~/.nvm/versions/node/v16.0.0/bin/npm
      Watchman: 4.9.0 - /opt/homebrew/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: DriverKit 21.2, iOS 15.2, macOS 12.1, tvOS 15.2, watchOS 8.3
    IDEs:
      Xcode: 13.2.1/13C100 - /usr/bin/xcodebuild
    npmPackages:
      @expo/webpack-config: ~0.16.15 => 0.16.15
      babel-preset-expo: 9.0.2 => 9.0.2
      expo: ^44.0.1 => 44.0.1
      react: 17.0.1 => 17.0.1
      react-dom: 17.0.1 => 17.0.1
      react-native: 0.64.3 => 0.64.3
      react-native-web: 0.17.5 => 0.17.5
    npmGlobalPackages:
      eas-cli: 0.41.1
      expo-cli: 5.0.3
    Expo Workflow: managed
```

### Reproducible demo

n/a

**Comment by pors** [comment]
Actually the problem occurs when upgrading "expo-asset": "8.4.4" to "8.4.5"

**Comment by jonsamp** [comment]
Hi @pors, I'd love to get the bottom of this issue. I tried to reproduce this issue with a new project, which you can clone/try here: https://github.com/jonsamp/expo-issue-15714/tree/main. I just created this repo with `expo init` (Expo CLI version 5.0.3)

In the README.md, I listed the stuff I did, along with screenshots of it working. Are you still seeing the issue with this minimum repro?

Can you share a reproducible reproduction repository of the issue? If so that would help me get to the bottom of this. You might also try running:

```bash
rm -rf node_modules yarn.lock // or nom equivalent
yarn install
```

To see if that resolves the issue.

**Comment by pors** [comment]
Thanks for the quick reply! I don't have the issue with your new project, but deleting and re-installing everything did solve it!

Should have done that myself first..

Thanks!

**Comment by rkstar** [comment]
is there a way to fix this **without** blowing away my lock file? that seems excessive and dangerous for production projects.

**Comment by pors** [comment]
> is there a way to fix this **without** blowing away my lock file? that seems excessive and dangerous for production projects.

In the end it doesn't help anyway. The warning is back for me. So I'll reopen again.

**Comment by DPflasterer** [comment]
Also running into this issue on the web. It looks like **getManifest2()** was defined in **PlatformUtils.js** but not **PlatformUtils.web.js**.

**Comment by jonsamp** [comment]
Thanks for tracking that down, opened a PR to add a stub for web. We'll get it merged and will release a new version.

**Comment by esamelson** [comment]
This is fixed in expo-asset@8.4.6 (or expo@44.0.5 if you only have expo-asset as a transitive dependency).
